apps/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,51 @@
-import { Component, OnInit } from '@angular/core';
+import { AsyncPipe } from '@angular/common';
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { map, Observable } from 'rxjs';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { Item } from '../../model/item.model';
+import { CardComponent } from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `<app-card
+    [items]="cityItems$ | async"
+    customClass="bg-light-blue"
+    (addNewItem)="addNewCity()">
+    <ng-container image>
+      <img src="assets/img/city.png" width="200px" />
+    </ng-container>
+    <ng-template #listItemRef let-item>
```

Comment:
this is it. Very nice. 
and just to be even more precise, You can call it let-city. 

And you can build your view from it and not in your component (which is not the place to do this)

<app-list-item>
  {{city.name}} {{city.country}}
</app-list-item>

and you can delete your map operaor l.34-37. 

cleaner 👍